### PR TITLE
Adding AWS Kinesis support

### PR DESCRIPTION
**Pull request by preved911**
Hello. We would like to use AWS Kinesis compatible target for reports transferring. Can you, please, look this PR?

### CODE REVIEW

**Comment by fjogeleit** [comment]
Hey, thanks for your PR. I will have a look on it.

File: charts/policy-reporter/Chart.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: |
   It creates Prometheus Metrics and can send rule validation events to different targets like Loki, Elasticsearch, Slack or Discord
 
 type: application
-version: 2.9.3
+version: 2.9.4
 appVersion: 2.6.1
```
Comment (fjogeleit) [inline charts/policy-reporter/Chart.yaml]: can you also prepare the new appVersion 2.6.2 here and in the values.yaml? which I will release when everything is merged

File: pkg/target/kinesis/kinesis.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package kinesis
```
Comment (fjogeleit) [inline pkg/target/kinesis/kinesis.go:1]: Can you also adopt a basic test? You can use the `s3_test.go` as example for it.

File: charts/policy-reporter/Chart.yaml (hunk 1 above)
Comment (preved911) [reply charts/policy-reporter/Chart.yaml]: Done

File: pkg/target/kinesis/kinesis.go (hunk 2 above)
Comment (preved911) [reply pkg/target/kinesis/kinesis.go:1]: Added test like `s3_test.go`

File: pkg/target/kinesis/kinesis_test.go [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+func Test_KinesisTarget(t *testing.T) {
+	t.Run("Send", func(t *testing.T) {
+		callback := func(body *bytes.Buffer, key string) {
+			report := new(bytes.Buffer)
+			json.NewEncoder(report).Encode(completeResult)
+
+			if body != report {
+				buf := new(bytes.Buffer)
+				buf.ReadFrom(body)
```
[the remark this answers is not in the document]
Comment (fjogeleit) [reply pkg/target/kinesis/kinesis_test.go]: @sonatype-lift ignore

File: pkg/target/kinesis/kinesis_test.go [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+	return c.err
+}
+
+var testCallback = func(body *bytes.Buffer, key string) {}
+
+func Test_KinesisTarget(t *testing.T) {
+	t.Run("Send", func(t *testing.T) {
+		callback := func(body *bytes.Buffer, key string) {
+			report := new(bytes.Buffer)
+			json.NewEncoder(report).Encode(completeResult)
```
[the remark this answers is not in the document]
Comment (fjogeleit) [reply pkg/target/kinesis/kinesis_test.go]: @sonatype-lift ignore

**Comment by fjogeleit** [comment]
Thanks for your contribution, I am currently working on another improvement, so the release can take a few days. I will also do some manual tests and add Kineses to the docs.

**Comment by fjogeleit** [comment]
I released v2.9.4 of the Helm Chart with your new push target.

**Pull request merged**